modules/ROOT/pages/do-not-use-to-find-info-with-new-search.adoc

```diff
@@ -0,0 +1,205 @@
+= Search for Assets
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Find assets and refine search results in Exchange using the following filters that are available from the asset portal:
+
+*  Search all assets field
+* Any type
+* Any organization
+* Any category
+* Any tag
+* Any Lifecycle state
+
+image:exchange-new-search.png[Exchange search experience]
+
+Search uses query language to search assets by tags, categories, custom fields, and lifecycle states. 
+
+== Search by Type
+
+Exchange supports the following asset types:
```

Comment:
Is it necessary to repeat the list from "Asset Types"?
Also, note that the types are listed in a different order here than in the Asset Types topic.